### DISCUSSION TITLE
请问作者还在更新嘛

### DISCUSSION THREAD

**Discussion by StudentWeis** [dillonzq/LoveIt#739] [Q&A]
如题，刚从 Hexo 转到 Hugo，觉得 Loveit 是个不错的主题，但是看到上一次 commiti 是七月份了，不知道作者大大还在更新嘛？

**Comment by geekswg** [comment]
目前很长时间没有更新了，作者可能比较忙，具体查看作者说明
[作者说明](https://github.com/dillonzq/LoveIt/issues/672)。
作者推荐的几个基于loveit主题还是不错的，可以用其他基于loveit开发的。
[预览地址](https://geekswg.js.cool/)
